src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:46217/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:46217/project/issues?id=sample&pullRequest=5919&issues=AWeEC_7qZzrxZ29MT9zd&open=AWeEC_7qZzrxZ29MT9zd)

[ISSUE_KEY]: AWeEC_7qZzrxZ29MT9zd